mos_tests/nova/conftest.py

```diff
@@ -0,0 +1,67 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+import tempfile
+
+import pytest
+from six.moves import configparser
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.yield_fixture
+def set_recl_inst_interv(env, request):
+    """Set 'reclaim_instance_interval' to 'nova.conf' on all nodes"""
+    nova_cfg_f = '/etc/nova/nova.conf'
+    nova_restart_ctrllr = 'service nova-api restart && sleep 3'
+    nova_restart_comput = 'service nova-compute restart && sleep 3'
+    interv_sec = request.param  # reclaim_instance_interval
+
+    logger.debug('In {} set "reclaim_instance_interval={}"'.format(
+        nova_cfg_f, interv_sec))
+    # take backup
+    _, backup_f = tempfile.mkstemp(prefix='novaconf_')
+    cmd = 'cp {0} {1}'.format(nova_cfg_f, backup_f)
+    # modify nova config file
+    for node in env.get_all_nodes():
+        with node.ssh() as remote:
+            remote.check_call(cmd)
+            with remote.open(nova_cfg_f, 'r') as f:
```

Comment:
Let's check that file exists (i'm not sure that ironic or mongo nodes contains it)